### PR TITLE
BZ#1794769 Verify the registry Pod has been moved to the infrastructure node

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1794769

### CODE REVIEW

**Comment by codyhoag** [comment]
@wzheng1 PTAL? Thanks!

Preview: https://verify-infra-node--ocpdocs.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#infrastructure-moving-registry_creating-infrastructure-machinesets

File: modules/infrastructure-moving-registry.adoc [hunk 1]
```diff
@@ -64,6 +64,34 @@ $ oc edit config/cluster
   nodeSelector:
     node-role.kubernetes.io/infra: ""
 ----
+
+. Verify the registry Pod has been moved to the infrastructure node.
++
```
Comment (wzheng1) [inline modules/infrastructure-moving-registry.adoc:69]: Better add more info like : Check the pod locates which node via oc get pods -o wide -n openshift-image-registry

Comment (codyhoag) [reply modules/infrastructure-moving-registry.adoc:69]: Thanks, @wzheng1. Just added that step and edited the existing steps slightly. Can you take another look? Thanks!

Comment (wzheng1) [reply modules/infrastructure-moving-registry.adoc:69]: Thanks! ack now : )

File: modules/infrastructure-moving-registry.adoc [hunk 2]
```diff
@@ -64,6 +64,21 @@ $ oc edit config/cluster
...
+.. Run the following command to identify the node where the registry Pod is
+located.
++
+----
+oc get pods -o wide -n openshift-image-registry
```
Comment (ahardin-rh) [inline modules/infrastructure-moving-registry.adoc:74]: $ oc

File: modules/infrastructure-moving-registry.adoc [hunk 3]
```diff
@@ -64,6 +64,21 @@ $ oc edit config/cluster
...
+----
++
+.. Confirm the node has the label you specified.
++
+----
+oc describe node <node_name>
```
Comment (ahardin-rh) [inline modules/infrastructure-moving-registry.adoc:80]: $ oc

File: modules/infrastructure-moving-registry.adoc [hunk 4]
```diff
@@ -64,6 +64,21 @@ $ oc edit config/cluster
   nodeSelector:
     node-role.kubernetes.io/infra: ""
 ----
+
+. Verify the registry Pod has been moved to the infrastructure node.
++
+.. Run the following command to identify the node where the registry Pod is
+located.
```
Comment (ahardin-rh) [inline modules/infrastructure-moving-registry.adoc:71]: located:

File: modules/infrastructure-moving-registry.adoc [hunk 5]
```diff
@@ -64,6 +64,21 @@ $ oc edit config/cluster
...
++
+----
+oc get pods -o wide -n openshift-image-registry
+----
++
+.. Confirm the node has the label you specified.
```
Comment (ahardin-rh) [inline modules/infrastructure-moving-registry.adoc:77]: specified:

**Comment by ahardin-rh** [comment]
@codyhoag Just some minor comments from me! :+1:

**Comment by codyhoag** [comment]
Thanks, @ahardin-rh!

**Pull request merged**